plugins/server-stats/api/jobs/stats.js

```diff
@@ -141,6 +141,17 @@ class StatsJob extends job.Job {
                                 });
 
                                 Countly.userData.save();
+
+                                Countly.fetch_remote_config(null, null, (e, conf) => {
+                                    if (err) {
+                                        log.e('Failed to fetch remote config', e);
+                                    }
+                                    else {
+                                        db.collection('plugins').updateOne({_id: 'plugins'}, {$set: {remoteConfig: conf || {}}}).catch(dbe => {
```

Comment:
@kanwarujjaval remote config is fetched during regular stats ping and stored in db, let me know if it won't work